jobs/runtime-runc/spec

```diff
@@ -28,4 +28,30 @@ properties:
       The name of the CNI plugin used for network IPAM.  For example, if you are using
       the `net-flannel` job on this node, use "flannel" as your `cni` value.
 
+  mirrors:
+    description: |
+       A list of additional mirror . Use this option with caution; it has the possibility to horribly break your deployment.
+       Each item in the list must have a name and an url and insecure_skip_verify_option.
+    default: []
+    example: |
+      - name: "docker.io"
```

Comment:
Is there a reason this can't be a map of keys, instead of a list of named objects?  It seems like it gets translated into a map on the containerd side, and having multiple objects with the same name might be bad.

Reply:
I'll do the change to convert the list to a map